src/components/video-editor/SettingsPanel.tsx

```
@@ -186,40 +235,59 @@ export function SettingsPanel({
+    (async () => {
       try {
         const resolved = await Promise.all(
-          WALLPAPER_RELATIVE_PATHS.map(async (path) => getRenderableAssetUrl(await getAssetPath(path)))
-        )
-        if (mounted) setWallpaperPreviewPaths(resolved)
+          WALLPAPER_RELATIVE_PATHS.map(async (path) =>
+            getRenderableAssetUrl(await getAssetPath(path)),
+          ),
+        );
+        if (mounted) setWallpaperPreviewPaths(resolved);
       } catch (err) {
```

Biome (2.4.6)

[error] This variable err is unused.

(lint/correctness/noUnusedVariables)